### DISCUSSION TITLE
Enhanced calendar integration: event invitation in confirmation emails

### DISCUSSION THREAD

**Discussion by EduNetEurope** [pretix/pretix#4363] [Ideas] [answered]
Users have the option to download an event iCal and then manually add it to their calendar, which is somewhat cumbersome.

It would be great if the event confirmation email included an event invitation. This would allow the event to be automatically added to the attendee's calendar without the need for manual intervention.

**Comment by pc-coholic** [comment, marked as answer]
If you check the the "Attach calendar files"-checkbox in Settings -> E-Mail -> Calendar Invites, then the iCal-files are indeed attached and automatically imported (if your mailclient is inclinded/configured to do so)

**Reply by EduNetEurope** [reply]
Thank you, @pc-coholic. I just received a pretix invitation from another organisation and had to add the event manually to my calendar (again). I will review our own settings. Thanks!
